dbt/adapters/databricks/relation.py

```diff
@@ -115,3 +125,13 @@ def matches(
     @classproperty
     def get_relation_type(cls) -> Type[DatabricksRelationType]:
         return DatabricksRelationType
+
+    def information_schema(self, view_name=None) -> InformationSchema:
```

Comment:
This is mostly copied from core, but I had to override so that it would use an information schema that used "`" for quoting, since InformationSchema inherits from BaseRelation.